flybywiresim/a32nx
# feat: efis overlay, fix info, and other huge changes

Fixes #1181

## Summary of Changes

- EFIS symbols are transmitted to the ND from the FMGC
  - `MAP PARTLY DISPLAYED` when too many words to send
  - Various other fixes as part of the implementation
- Waypoint, VOR/DME, NDB, and Airport overlay supported on ND (backed by temporary nearest facility impl. until FBW nav db available)
- Fix info partially implemented (radials and radii shown on ND, but no flightplan intercepts calculated)
- FA leg if no SID selected has correct bearing
- Runway loading code fixed
  - DER leg on SIDs works again
-  VOR reminder symbol not shown on ND for DME-only stations (bottom corners)
- TO waypoint data only shows when valid
- Speed constraints treated per the AMM for descent and approach
- Transition altitudes work correctly for constraint display on ND
- Various small fixes for ND

## Screenshots (if necessary)


![image](https://user-images.githubusercontent.com/9995998/134772135-ca79b97a-d05e-499c-ab0c-24973e99d9ce.png)


## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions


- Make sure EFIS filters work.
- Make sure EFIS filters cause `MAP PARTIALLY DISPLAYED` message in an area with a lot of waypoints.
- Make sure FIX INFO feature works properly.
- Make sure there are no other regressions in ND symbology.


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
